### PR TITLE
feat: rewrite the reward related

### PR DESCRIPTION
**Pull request by FMint**

### CODE REVIEW

File: humanoidverse/config/rewards/motion_tracking/reward_motion_tracking_dm_2real.yaml [hunk 1]
```diff
@@ -9,28 +9,30 @@ rewards:
...
-      penalty_slippage: -1.0
+      ##penalty_feet_ori: -2.0
+      ##feet_heading_alignment: -0.1
+      ##penalty_slippage: -1.0
 
       limits_dof_pos: -10.0
       limits_dof_vel: -5.0
-      limits_torque: -5.0
+      limits_torque: -0.1
       termination: -200.0
```
Comment (kvkcon) [inline humanoidverse/config/rewards/motion_tracking/reward_motion_tracking_dm_2real.yaml:36]: discard 

we do dev test using `reward_motion_tracking_basic.yaml` first

File: humanoidverse/envs/legged_base_task/legged_robot_base.py [hunk 2]
```diff
@@ -243,7 +246,25 @@ def delta_step(self,delta_action):
         return self.obs_buf_dict, self.rew_buf, self.reset_buf, self.extras
 
     def set_sim_state_from_real_data(self,s_real):
-        self.simulator.all_
+        env_ids = torch.arange(self.num_envs,device=self.device)
+
+        #creatr root_states tensor [num_envs,13]
+        root_states = torch.zeros((self.num_envs,13),device=self.device)
+        root_states[:,0:3]=s_real["base_pos"]
```
Comment (kvkcon) [inline humanoidverse/envs/legged_base_task/legged_robot_base.py]: rm

File: humanoidverse/envs/legged_base_task/legged_robot_base.py [hunk 3]
```diff
@@ -243,7 +246,25 @@ def delta_step(self,delta_action):
...
+        root_states[:,3:7]=s_real["base_quat"]
+        root_states[:,7:10]=s_real["base_lin_vel"]
```
Comment (kvkcon) [inline humanoidverse/envs/legged_base_task/legged_robot_base.py]: rm base_lin_vel
